testing/test_fot

```diff
@@ -58,17 +58,15 @@ $YQ '.dps."nz-kiwi-t2sw1".interfaces."1".native_vlan' $BEHAVIORAL_CONFIG | tee -
 $YQ '.dps."nz-kiwi-t2sw2".interfaces."1".native_vlan' $BEHAVIORAL_CONFIG | tee -a $TEST_RESULTS
 
 # Start DHCP client on faux-1 and faux-2
-docker exec forch-faux-1 dhclient -r &
-docker exec forch-faux-2 dhclient -r
 docker exec forch-faux-1 dhclient &
```

Comment:
My comment was more about these -- dhclient should be run by a command line arg to the faux container startup, unless that's very difficult for some reason.

Reply:
Added faux_dhcp option for faux-1 and faux-2 in bin/setup_stack:145-146. This will start dhcp client during the faux container startup.